spec/ddtrace/contrib/sidekiq/support/helper.rb

```diff
@@ -51,6 +52,9 @@ def expect_in_sidekiq_server(duration: 2, &expectations)
       # spans created in the forked process, the exit status will be 0 even if our expectations fail.
       # Instead, look at `STDERR`, ignoring warnings.
       fork_expectations: proc do |status:, stdout:, stderr:|
+        stdout = Datadog::Core::Utils.utf8_encode(stdout) if stdout
+        stderr = Datadog::Core::Utils.utf8_encode(stderr) if stderr
```

Comment:
Why were these added?

Reply:
The tests were complaining that ASCII /UTF8 outputs were not compatible/comparable. I don't why this started to happen but this did fix the problem. (Maybe because it had some other error that triggered ASCII output?)